src/minimega/vnc_cli.go

```diff
@@ -27,72 +28,106 @@ If playback is selected, the specified file (created using vnc record) will be
 read and processed as a sequence of time-stamped mouse/keyboard events to send
 to the specified VM.`,
 		Patterns: []string{
-			"vnc",
-			"vnc <kb,fb> <record,> <host> <vm id or name> <filename>",
-			"vnc <kb,fb> <norecord,> <host> <vm id or name>",
-			"vnc <playback,> <host> <vm id or name> <filename>",
-			"vnc <noplayback,> <host> <vm id or name>",
+			"vnc <kb,fb> <record,> <vm name> <filename>",
+			"vnc <kb,fb> <norecord,> <vm name>",
+			"vnc <playback,> <vm name> <filename>",
+			"vnc <noplayback,> <vm name>",
```

Comment:
While we're changing things... should we swap the order of `<kb,fb>` and `<record>`?